screencap.py

```diff
@@ -119,46 +134,46 @@ def retranslateUi(self, mainWindow):
         self.actionmvk.setText(_translate("mainWindow", "mvk"))
 
     def clicked_format(self, extension):
-        f = open('memory_screenshot.txt','r')
-        lst = f.read().split(',')
-        f.close()
-        f = open('memory_screenshot.txt','w')
-        lst[1] = extension
-        f.write('{0},{1},{2}'.format(lst[0],lst[1],lst[2]))
-        f.close()
+        previous = data['image']['extension']
+        data['image']['extension'] = extension
+        if previous != data['image']['extension']:
+            with open('memory.json', 'w') as f:
+                f.seek(0)
+                f.write(json.dumps(data, indent=2))
+                f.truncate()
 
     def clicked_folder(self):
-        self.screenshot_path = QtWidgets.QFileDialog.getExistingDirectory(None, 'Select Folder','C:/Users')
-        f = open('memory_screenshot.txt','r')
-        lst = f.read().split(',')
-        f.close()
-        f = open('memory_screenshot.txt','w')
-        lst[2] = self.screenshot_path
-        f.write('{0},{1},{2}'.format(lst[0],lst[1],lst[2]))
-        f.close()
-
+        previous = data['image']['path']
+        screenshot_path = QtWidgets.QFileDialog.getExistingDirectory(None, 'Select Folder','C:/Users')
+        data['image']['path'] = screenshot_path
+        if previous != data['image']['path']:
```

Comment:
The same comments as for clicked_format allies here. By the way, the code for saving the config to the file is the same in both functions, what about transferring it into a separate function?